### PR TITLE
remove all docs references to baselines

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

File: configure/topology/propagation.md [hunk 1]
```diff
@@ -1,44 +0,0 @@
----
-description: Propagate health states from dependencies to dependents
----
```
Comment (konstantintatarnikov) [inline configure/topology/propagation.md:3]: Why Propagation function doc has been removed?

File: develop/developer-guides/functions.md [hunk 2]
```diff
@@ -1,41 +0,0 @@
-# Overview
-
-Functions in StackState are predefined scripts that transform input into an output. They are called by StackState on-demand. For example, when a component changes state, new telemetry flows in ora user triggers an action. Advanced users can develop their own functions to customize StackState. These functions can then be exported and packaged with a custom StackPack. Read more about it in [how to create StackPacks](../../stackpacks/about-stackpacks.md).
```
Comment (konstantintatarnikov) [inline develop/developer-guides/functions.md:3]: hmm , this is just general information about functions - not about baselines, should we keep it?

File: configure/topology/propagation.md (hunk 1 above)
Comment (gggina) [reply configure/topology/propagation.md:3]: this was an old, unused page.
it is now /develop/developer-guides/custom-functions/propagation-functions.md

File: develop/developer-guides/functions.md (hunk 2 above)
Comment (gggina) [reply develop/developer-guides/functions.md:3]: this is an archived page. it was not used

**Review by konstantintatarnikov — APPROVED** [review]

**Pull request merged**